### DISCUSSION TITLE
Dispatch Component Not Visible on frontend

### DISCUSSION THREAD

**Discussion by aravindkumar1982** [learndevopsonline/learndevopsonline#1408] [Q&A] [answered]
Hello @r-devops,

Hope you are all well. I am trying to set up dispatch component. I am not able to find this on the front end

To connect:

![Screenshot 2023-09-04 at 15 50 21](https://github.com/learndevopsonline/learndevopsonline/assets/139839369/a7f8834d-6a0a-4ee0-98ec-e5bf2285b578)

![Screenshot 2023-09-04 at 15 45 28](https://github.com/learndevopsonline/learndevopsonline/assets/139839369/e2f0383f-a0ed-489e-9d94-724039e14b63)

Kindly note: For frontend login
User name: kumar
passwd: kumar

I did run the dispatch code, I did not see any errors.

![Screenshot 2023-09-04 at 15 47 27](https://github.com/learndevopsonline/learndevopsonline/assets/139839369/32f64068-68b1-4f8a-878c-46f3b2dda4d2)

Hope this helps. 

Thanks for your time.

Kind regards,

Aravind

**Comment by sreedharm07** [comment, marked as answer]
hi @aravindkumar1982, dispatch will not appear in frontend ui, log in to dispatch server and see logs (systemctl restart dispatch ; tail -f /var/log/messages) you will find the shipping details in logs. in logs if it shows ur shipping details your dispatch server is working fine.

**Reply by aravindkumar1982** [reply]
Thanks Raghu
